shared/js/content-scripts/breakage-flow.js

```diff
@@ -0,0 +1,403 @@
+        })
+        const isBrokenButton = makeButton('No thanks', 'cancelMode')
+        isBrokenButton.style.cssText += styles.modalButton
+        isBrokenButton.addEventListener('click', function isBroken () {
+            document.body.removeChild(modalContainer)
+        })
+
+        buttonRow.appendChild(notBrokenButton)
+        buttonRow.appendChild(isBrokenButton)
+        modalContent.appendChild(buttonRow)
+
+        modal.appendChild(modalContent)
+        modalContainer.appendChild(modal)
+
+        return modalContainer
+    }
+
+    function makeConfirmFixModal () {
+        const modalContainer = document.createElement('div')
+        modalContainer.style.cssText = styles.modalContainer
+        const styleElement = document.createElement('style')
+        styleElement.textContent = styles.fontStyle
+        modalContainer.appendChild(styleElement)
+
+        const modal = document.createElement('div')
+        modal.style.cssText = styles.modal
+
+        const modalContent = document.createElement('div')
+        modalContent.style.cssText = styles.modalContent
+
+        const closeContainer = document.createElement('div')
+        closeContainer.style.cssText = styles.closeContainer
+
+        const closeButton = document.createElement('div')
+        closeButton.style.cssText = styles.closeButton
+        closeButton.textContent = 'x'
+        closeButton.addEventListener('click', function () {
+            document.body.removeChild(modalContainer)
+        })
+
+        closeContainer.appendChild(closeButton)
+        modalContent.appendChild(closeContainer)
+
+        const title = document.createElement('div')
+        title.style.cssText = styles.modalContentTitle
+        title.textContent = 'Did disabling Privacy Protection fix the problem you experienced?'
+
+        modalContent.appendChild(title)
+
+        // Buttons
+        const buttonRow = document.createElement('div')
+        buttonRow.style.cssText = styles.buttonRow
+        const notBrokenButton = makeButton('No', 'lightMode')
+        notBrokenButton.style.cssText += styles.modalButton
+        notBrokenButton.addEventListener('click', function notBroken () {
+            siteNotFixed(modalContainer, modalContent)
+        })
+        const isBrokenButton = makeButton('Yes', 'lightMode')
+        isBrokenButton.style.cssText += styles.modalButton
+        isBrokenButton.addEventListener('click', function isBroken () {
+            confirmSiteFixed(modalContainer, modalContent)
+        })
+
+        buttonRow.appendChild(notBrokenButton)
+        buttonRow.appendChild(isBrokenButton)
+        modalContent.appendChild(buttonRow)
+
+        modal.appendChild(modalContent)
+        modalContainer.appendChild(modal)
+
+        return modalContainer
+    }
+
+    function confirmSiteFixed (modalContainer, modalContent) {
+        // Remove modal content
+        while (modalContent.childNodes.length > 1) {
```

Comment:
Can you make this a method? Alternatively just rebuild the modal DOM via a function? It would probably better to delete everything when it's not required.